docs/scripts/update-contributors.js

```diff
@@ -182,12 +182,19 @@ export const OPENAPI_FETCH_CONTRIBUTORS = [
   ]),
 ];
 
+export const OPENAPI_REACT_QUERY_CONTRIBUTORS = [...new Set(["drwpow", "kerwanp"])];
+
 async function main() {
-  const total = [...OPENAPI_TS_CONTRIBUTORS, OPENAPI_FETCH_CONTRIBUTORS].length;
+  const total = [...OPENAPI_TS_CONTRIBUTORS, OPENAPI_FETCH_CONTRIBUTORS, OPENAPI_REACT_QUERY_CONTRIBUTORS].length;
   let i = 0;
   await Promise.all(
-    ["openapi-typescript", "openapi-fetch"].map(async (repo) => {
-      const userlist = repo === "openapi-fetch" ? OPENAPI_FETCH_CONTRIBUTORS : OPENAPI_TS_CONTRIBUTORS;
+    ["openapi-typescript", "openapi-fetch", "openapi-react-query"].map(async (repo) => {
```

Comment:
👍 Update looks good. As you probably found out, this scraper just `429`s. Probably needs to get updated to use the actual REST API with an auth token, but I just haven’t put the time in to do that

Reply:
I didn't get the 429 on my side but this definitely need to be fixed as the list will (hopefully) increase. I'll have some free time this week, I'll do a PR for that